test/identityManager.js

```diff
@@ -56,6 +56,8 @@ contract('IdentityManager', (accounts) => {
   let user4
   let user5
   let nobody
+  let data
+  let testNum
```

Comment:
You can just declare these variables inside the specific test. Same thing for the metaIdentityManager test.